### PR TITLE
Initializing project structure

### PR DESCRIPTION
**Pull request by jdrgoms**
##### The contributions were made:
- add Jekyll structure version 3.1
- created .gitignore
- created README and inserted instructions for use

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: Your awesome title
```
Comment (raphamorim) [inline _config.yml]: @jjaderg can you configure following origamis settings and remove all comments from this file?

File: _posts/2016-02-11-welcome-to-jekyll.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (raphamorim) [inline _posts/2016-02-11-welcome-to-jekyll.markdown]: I think we might not need this page.

File: feed.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (raphamorim) [inline feed.xml]: I think we might not neet this page too.
You can remove all blog entries also :)

File: _sass/_base.scss [hunk 4]
```diff
@@ -0,0 +1,206 @@
+/**
```
Comment (raphamorim) [inline _sass/_base.scss]: Why scss?

File: _config.yml (hunk 1 above)
Comment (jdrgoms) [inline _config.yml]: @raphamorim yes i can!

File: _posts/2016-02-11-welcome-to-jekyll.markdown (hunk 2 above)
Comment (jdrgoms) [inline _posts/2016-02-11-welcome-to-jekyll.markdown]: I agree ;)

File: _sass/_base.scss (hunk 4 above)
Comment (jdrgoms) [inline _sass/_base.scss]: It's a Jekyll standard :P. But I have no preference among CSS pre processors and their extensions. However I use more Stylus. :)

Comment (raphamorim) [inline _sass/_base.scss]: Yes I know that.
I think to start, it'll be better to use pure CSS to reduce processes and the amount of knowledge to contribute.

I want keep it simple: Static pages created after run a simple command and nothing beyond it.
What do you think?

Comment (jdrgoms) [inline _sass/_base.scss]: I think valid, taking into account the real objective of the project. The simpler the better!

**Comment by raphamorim** [comment]
:+1: 
@jjaderg can you squash your commits to I merge it?

**Comment by raphamorim** [comment]
Merged in another branch, which has your commits :)

**Pull request closed without merging**